arches_querysets/models.py

```diff
@@ -38,6 +38,16 @@
 
 
 logger = logging.getLogger(__name__)
+NOT_PROVIDED = object()
+
+
+class AliasedData:
+    def __init__(self, **kwargs):
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    def __repr__(self):
+        return f"<AliasedData: {vars(self)}"
 
 
 class SemanticResource(ResourceInstance):
```

Comment:
I may look into adding some magic methods here like `__getattr__()` that automatically try to grab the item off of `.aliased_data`, but that's just saving keystrokes, so not for this this first iteration.